### PR TITLE
Fix docs enrolment spelling - to queens english

### PR DESCRIPTION
**Pull request by JustinAClarke**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -8,6 +8,11 @@
     <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
     <link rel="icon" type="image/png" href="./favicon.png">
     <script type="module" src="https://unpkg.com/rapidoc/dist/rapidoc-min.js"></script>
+    <script type="text/javascript">
```
Comment (armatronic) [inline docs/index.html:11]: Isn't this file auto-generated by the build tool?

File: openapi/components/schemas/assessment-feedback-list.yaml [hunk 2]
```diff
@@ -30,15 +30,34 @@ properties:
               example: dueWork
             commonAssessment:
               type: boolean
-            weight:
-              type: string
-              example: 1.00
+            weighted:
```
Comment (armatronic) [inline openapi/components/schemas/assessment-feedback-list.yaml:33]: Interesting: we're only returning whether an assessment is weighted, not its actual weight.

There might not be much point to returning an assessment's weighting until we provide an easy method for fetching assessments by term, though.

File: docs/index.html (hunk 1 above)
Comment (JustinAClarke) [reply docs/index.html:11]: not anymore, that was using redocly, only the `docs/dist.yaml` is compiled now

File: openapi/components/schemas/assessment-feedback-list.yaml (hunk 2 above)
Comment (JustinAClarke) [reply openapi/components/schemas/assessment-feedback-list.yaml:33]: thoughts?
return weight value, or leave as boolean?

**Pull request merged**